src/task1/Test.java

```diff
@@ -0,0 +1,33 @@
+package task1;
+
+import java.util.*;
+
+public class Test {
+    public static void main(String[] args) {
+        ArrayList<Integer> integerList = new ArrayList<>();
+
+        Random random = new Random();
+        for (int i=0; i<100; i++){
+            int randomValue = random.nextInt(41)-20;
+            integerList.add(randomValue);
+        }
+        System.out.println("Original Collection:");
+        System.out.println(integerList);
+
+        HashSet<Integer> uniqueCollection = new HashSet<>(integerList);
+        ArrayList<Integer> collectionWithoutDuplicates = new ArrayList<>(uniqueCollection);
```

Comment:
What is the reason to use one more collection. In HashSet all elements are unique